Dockerfile

```diff
@@ -0,0 +1,41 @@
+# Dockerfile for cBioPortal
+FROM elementolab/jamm:1.0
+MAINTAINER Alexandros Sigaras <als2076@med.cornell.edu>
+#===========================#
+# Clone cBioPortal			#
+#===========================#
+#RUN git clone https://github.com/cBioPortal/cbioportal.git; \
+ADD / /cbioportal
+RUN	echo "export PORTAL_HOME=/cbioportal" >> /root/.bash_profile;
```

Comment:
An `ENV` command would probably be a lot clearer, especially since practically every single command run in a container using this image will need this variable, and they probably will not want to run a bash shell along with them to explicitly parse .bash_profile.